From f283a6b126356fdc7dcb7c803b65c5cb473b78d9 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Tue, 17 Jul 2018 15:30:48 -0500
Subject: [PATCH] Use netcoreapp3.0 for compatibility with everything else.

This patch needs further research.  I believe it will become unnecessary
after core-setup/cli/toolset/core-sdk have been synced.
---
 Directory.Build.props | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Directory.Build.props b/Directory.Build.props
index 108ebda8a..2445f7fc3 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -33,9 +33,9 @@ tools\TestAssetsDependencies\TestAssetsDependencies.csproj
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
     <DisableImplicitPackageTargetFallback>true</DisableImplicitPackageTargetFallback>
 
-    <CliTargetFramework>netcoreapp2.2</CliTargetFramework>
+    <CliTargetFramework>netcoreapp3.0</CliTargetFramework>
     <!-- We only need this until we get a stage0 with a 2.2 SDK. -->
-    <NETCoreAppMaximumVersion>2.2</NETCoreAppMaximumVersion>
+    <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
   </PropertyGroup>
 
   <Import Project="build/BranchInfo.props" />
-- 
2.14.1

